resources/helm/v2.1/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -253,5 +253,15 @@ spec:
       - name: extracacerts
         configMap:
           name: pilot-jwks-extra-cacerts-{{ .Values.revision | default "default" }}
+  {{- end }}
```

Comment:
this change will either have to go into maistra/istio repo (because that's where we pull these charts from) or into a sed script in the build/ directory